src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortEngine.java

```diff
@@ -163,8 +167,14 @@ public void traverse() {
                     logger.debug("using storage api with local sort");
                 }
                 logger.debug("Initializing Reader");
-                final StorageAPIAvroReader storageAPIAvroReader = new StorageAPIAvroReader(cohortTableRef, rowRestriction, projectID);
-                createVariantsFromUngroupedTableResult(storageAPIAvroReader, fullVqsLodMap, fullYngMap, noFilteringRequested);
+                if (cohortAvroFileName == null){
+                    final StorageAPIAvroReader StorageAPIAvroReader = new StorageAPIAvroReader(cohortTableRef, rowRestriction, projectID);
```

Comment:
lower case the java variablename `final StorageAPIAvroReader storageAPIAvroReader`